### ISSUE TITLE
macOS 10.14 (Mojave) breaks StubbyManager GUI

### ISSUE THREAD

**Issue by tyrotex** [Sinodun/stubby_manager_gui#2]
Thanks for the GUI app. It is very useful.

Unfortunately, the GUI will no longer run for me since I upgraded to the latest OS (10.14.1).
Stubby is still running, but the GUI interface crashes when I try to start it.

If I run the StubbyManager executable in /Applications/StubbyManager.app/Contents/MacOS/ in a Terminal, the errors are:

---

QMetaObject::connectSlotsByName: No matching signal for on_testerProcess_finished(int,QProcess::ExitStatus)
runtask.cpp : RunHelperTask Auth create  true
Assertion failed: (oss == errAuthorizationSuccess), function RunHelperTask, file runtask.cpp, line 61.
Abort trap: 6

---

It seems like Mojave has introduced some different security measures which block the assertion.

In the meantime, I am able to restart stubby (it gets confused sometimes and DNS queries fail) by using launchctl to unload/load /Library/LaunchDaemons/org.getdns.stubby.plist.

But, I do hope the GUI can be easily fixed sometime soon.

**Comment by banburybill** [comment]
I don't have a Mac handy to test this right now, but you may find that giving Stubby Manager accessibility under Settings/Security and Privacy works around this.

On 11 November 2018 04:28:32 GMT+00:00, John Bluhdorn <notifications@github.com> wrote:
>Unfortunately, the GUI will no longer run for me since I upgraded to
>the latest OS (10.14.1).
>Stubby is still running, but the GUI interface crashes when I try to
>start it.
-- 
Sent from a computer I can lift.

**Comment by tyrotex** [comment]
Thanks for the suggestion.

Unfortunately, I've already tried that and it makes no difference.

**Comment by banburybill** [comment]
This morning I've tried Stubby Manager on Mojave. And rather puzzlingly it's working fine for me. And I don't have Stubby Manager in my accessibility list.

Is there any possibility that the authorisation rights need to be reapplied?

**Comment by tyrotex** [comment]
That is strange. 
Before I put in this issue, I've tried running SM on two different Mojave systems, with the same failure.
However, in both cases, SM was installed through a  Migration Assistant transfer from previous High Sierra systems.
However, I've just tried re-installing from the original .pkg and it's now working fine.
Thanks for responding. My apologies for wasting everyone's time.
I'll try to close this (non) issue now.
